Blogpost for PathoGFAIR preprint on BioRxiv (#2682)
galaxyproject/galaxy-hub: content/news/2024-07-08-pathogfair-preprint/index.md

## Training Material

To assist users in understanding and using the workflows, we have provided extensive training material. This includes tutorials, documentation, and guidelines available via the Galaxy Training Network [GTN](https://bit.ly/pathogen-tuto).

The training has been teached in multiple workshops and conferences' training sessions, the [first time](https://galaxyproject.org/events/2023-02-02-foodborne-pathogen-detection-workshop/) was in [FHNW in Muttenz Switzerland](https://galaxyproject.org/news/2023-03-21-foodborne-training/), then in following **High-Throughput Data Analysis with Galaxy** workshops the latest is going to be on [July 26th 2024](https://galaxyproject.org/events/2024-07-22-galaxy-workshop-freiburg/). The training also took part in the Galaxy Community Conference [GCC 2024](https://galaxyproject.org/events/gcc2024/training/microbiome-analysis).

## Funding

> PathoGFAIR is the main [project](https://galaxyproject.org/news/2021-12-08-pathogen-detection-eosc-life-grant/) output, which was under the fund of [EOSC life industry call 2021](https://www.eosc-life.eu/industrycall/).

Reviewer: ```suggestion
The PathoGFAIR project was funded via [EOSC life industry call 2021](https://www.eosc-life.eu/industrycall/) ([news about the award](content/news/2021-12-08-pathogen-detection-eosc-life-grant/) 
```

---

> title: "New Preprint: PathoGFAIR: a collection of FAIR and adaptable (meta)genomics workflows for (foodborne) pathogens detection and tracking"

Reviewer: ```suggestion
title: "New Preprint: PathoGFAIR, a collection of FAIR and adaptable (meta)genomics workflows for (foodborne) pathogens detection and tracking"
```

## Training Material

To assist users in understanding and using the workflows, we have provided extensive training material as a [tutorial via the Galaxy Training Network (GTN)](https://bit.ly/pathogen-tuto).

This tutorial has been used as training support in multiple workshops and conferences' training sessions:

> 1. The [first time](/content/events/2023-02-02-foodborne-pathogen-detection-workshop/) in FHNW in Muttenz, Switzerland with [highly positive feedback](/content/news/2023-03-21-foodborne-training/)

Reviewer: ```suggestion
1. A [1-day workshop early 2023](/content/events/2023-02-02-foodborne-pathogen-detection-workshop/) in FHNW in Muttenz, Switzerland that received [highly positive feedback](/content/news/2023-03-21-foodborne-training/)
```

> 2. As a session in several **High-Throughput Data Analysis with Galaxy** workshops delivered in Freiburg with the next iteration will be on [July 26th 2024](/content/events/2024-07-22-galaxy-workshop-freiburg/).

Reviewer: ```suggestion
2. A 1-day session in several **High-Throughput Data Analysis with Galaxy** workshops delivered in Freiburg with the next iteration will be on [July 26th 2024](/content/events/2024-07-22-galaxy-workshop-freiburg/).
```

> 3. As a workshop during the Galaxy Community Conference [GCC 2024](/content/events/gcc2024/training/microbiome-analysis).

Reviewer: ```suggestion
3. A 2-hour workshop during the Galaxy Community Conference [GCC 2024](/content/events/gcc2024/training/microbiome-analysis).
```

## Funding

The PathoGFAIR project was funded via [EOSC life industry call 2021](https://www.eosc-life.eu/industrycall/) ([news about the award](content/news/2021-12-08-pathogen-detection-eosc-life-grant/))

> <embed src="2024.06.26.600753v1.full.pdf" width="100%" height="700" type='application/pdf'>

Reviewer: ```suggestion
<embed src="https://www.biorxiv.org/content/10.1101/2024.06.26.600753v1.full.pdf" width="100%" height="700" type='application/pdf'>
```
